charts/opensearch-cluster/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for OpenSearch Cluster
 type: application
 
 ## The opensearch-cluster Helm Chart version
-version: 3.0.0
+version: 3.0.1
```

Comment:
Once this PR is merged, The `labels: {}` will only be in next release of operator https://github.com/opensearch-project/opensearch-k8s-operator/issues/1011, we should not increment the opensearch-cluster Helm Chart and release it.

We can increment the helm chart and release it, if it has fixed with the current released operator or helm side enhancements.

Reply:
So I would suggest we decouple the operator and opensearch-cluster Helm Chart changes.

Reply:
Good point, I will split this into a separate PR!

Reply:
Thanks let me know once this version increment is removed from the PR, we can merge this

Reply:
Removed